build.sbt

```diff
@@ -5,16 +5,48 @@ name := "commercetools-sunrise-cms"
 
 organization := "io.commercetools.sunrise"
 
+/**
+ * PROJECT DEFINITIONS
+ */
+
 lazy val `commercetools-sunrise-cms` = (project in file("."))
   .configs(IntegrationTest)
+  .aggregate(`cms-common`, `cms-contentful`)
   .settings(javaUnidocSettings ++ commonSettings ++ commonTestSettings : _*)
 
+lazy val `cms-common` = project
+  .configs(IntegrationTest)
+  .settings(commonSettings ++ commonTestSettings : _*)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.apache.commons" % "commons-lang3" % "3.0",
```

Comment:
I should start double checking the dependencies :D
Sorry, I just want to finish this already :(